widget/form.go

```diff
@@ -39,7 +40,18 @@ func (f *Form) Move(pos fyne.Position) {
 
 // MinSize returns the smallest size this widget can shrink to
 func (f *Form) MinSize() fyne.Size {
```

Comment:
Thanks for this. Unfortunately somehow this seems not idea.
MinSize() is a "getter" method in that it returns data and so should have no side effects.

Maybe this is a wider issue but I think it would be fair to say that either the widget should cause a layout when it knows it's needed through the line addition, or that the containers should be re-laying out when they see a child item's minSize is changed.